apis/v1alpha1/httproute_types.go

```diff
@@ -112,9 +135,9 @@ type HTTPRouteRule struct {
 	// path match on "/", which has the effect of matching every
 	// HTTP request.
 	//
-	// +optional
 	// +kubebuilder:default={{path:{ type: "Prefix", value: "/"}}}
-	Matches []HTTPRouteMatch `json:"matches"`
+	// +kubebuilder:validation:MaxItems=8
```

Comment:
// +kubebuilder:validation:MinItems=1 to match other required lists.

Reply:
Do we need a minItems here if we already have a default?

Reply:
Probably not.